package.json

```diff
@@ -89,7 +89,7 @@
     "@types/dompurify": "^2.3.1",
     "@types/jasmine": "^3.8.2",
     "@types/jasminewd2": "2.0.8",
-    "@types/node": "~12.12.6",
+    "@types/node": "^15.6.1",
```

Comment:
should it be 14?

Reply:
When I put it to 14 it fails haha, I also thought it was like that but actually there's a table that maps typescript versions with node versions haha
https://stackoverflow.com/questions/64924337/i-am-getting-an-error-when-compiling-angular-6-project/65026470#65026470

Reply:
Can take a look at the errors here
https://github.com/CATcher-org/CATcher/runs/7536203036?check_suite_focus=true